app/decorators/site_decorator.rb

```diff
@@ -34,4 +34,13 @@ def page_title(posts)
       tagline
     end
   end
+
+  def serials_page_title(serials)
```

Comment:
Can you create `SerialsDecorator` and move `serials_page_title` to it?
The `serials_page_title` method doesn't use the attributes of `Site`, so this method shouldn't exist on `SiteDecorator`.